apps/02RockPaperScissors.js

```diff
@@ -9,6 +9,8 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
```

Comment:
I had a little trouble getting this one, but it looks like you got it right. :) I started off by putting .toLowerCase on all of my hand1's and hand2's. Oops.